changelogs/fragments/5048-iso-customize-getfiles.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+  - iso_customize - add new iso_customize.py module file in files folder
+  - iso_get_files - add new iso_get_files.py module file in files folder
```

Comment:
Please remove this. New modules should not have a changelog fragment. (Generally see `CONTRIBUTING.md` for information on how to add new modules.)

Reply:
Thanks. Done